search-app/resources/templates/search-base.html

```diff
@@ -2,5 +2,5 @@
 
 {% block app-nav %}
   <li><a href="{{ base-url }}site/docs/search">Documentation</a></li>
-  <li><a href="{{ base-url }}site/collections/directory">Directory</a></li>
+  <li><a href="{{ base-url }}site/collections/directory/eosdis">Directory</a></li>
```

Comment:
Good call. Just need to remember to revert this back if any other groupings of collections are added.